internal/commands/app/describe_test.go

```diff
@@ -0,0 +1,380 @@
+package app
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/atlas"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/local"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+
+	"github.com/Netflix/go-expect"
+)
+
+func TestDescribeInputsResolve(t *testing.T) {
+	t.Run("should set app if in app directory", func(t *testing.T) {
+		profile, teardown := mock.NewProfileFromTmpDir(t, "app-describe-test")
+		defer teardown()
+
+		assert.Nil(t, ioutil.WriteFile(
+			filepath.Join(profile.WorkingDirectory, local.FileRealmConfig.String()),
+			[]byte(`{"config_version":20210101,"app_id":"test-app-abcd","name":"test-app"}`),
```

Comment:
i cant help it, but our client app ids have a 5-character suffix to them
```suggestion
			[]byte(`{"config_version":20210101,"app_id":"test-app-abcde","name":"test-app"}`),
```